consensus/istanbul/core/roundstate.go

```diff
@@ -49,6 +49,7 @@ type RoundState interface {
 	AddParentCommit(msg *istanbul.Message) error
 	SetPendingRequest(pendingRequest *istanbul.Request) error
 	SetProposalVerificationStatus(proposalHash common.Hash, verificationStatus error)
+	SetBlockConsensusAndProcessResult(sealHash common.Hash, blockProcessResult *istanbul.BlockConsensusAndProcessResult)
```

Comment:
After creating a new type for the `BlockProcessResult`, this function name should be renamed accordingly.